types/app.d.ts

```diff
@@ -26,6 +26,8 @@ declare namespace Realm {
     // See https://stackoverflow.com/a/51114250 on why we're importing the BSON types like this
     type ObjectId = import("bson").ObjectId;
     type Binary = import("bson").Binary;
+    type Long = import("bson").Binary;
```

Comment:
Why import `Binary` three times under different names?

Reply:
Typo in last commit. Fixed by https://github.com/realm/realm-js/pull/3004/commits/76aae2b4718388600b898ec8d13202119db8089e